[docs] [yba] Create custom queries from the Prometheus window from the Performance metrics page (#21905)
yugabyte/yugabyte-db: docs/content/preview/yugabyte-platform/alerts-monitoring/anywhere-metrics.md

Click the refresh button to restore the default time range.

> You can also create custom queries and run it against any graph. To do this, from the **Metrics** page, click the Prometheus icon at the top right corner of any metrics graph, and paste your query in the Prometheus query editor. For example, to view the per-table p99 latency metrics, you can run the following PromQL query which uses the `ql_read_latency_sum` and `ql_read_latency_count` functions as follows:

Reviewer: ```suggestion
You can create custom PromQL queries and run them against any graph. To do this, from the **Metrics** page, click the Prometheus icon at the top right corner of any metrics graph to display the Prometheus window, then paste your query in the Prometheus query editor.

For example, to view the per-table p99 latency metrics, you can run the following PromQL query which uses the `ql_read_latency_sum` and `ql_read_latency_count` functions as follows:
```

```sql
label_replace(
   quantile_over_time(0.99, ((sum(rate(ql_read_latency_sum{node_prefix="<universe_name>", export_type="tserver_export", namespace_name!="system_platform"}[36s])) by (table_name))
   /
   (sum(rate(ql_read_latency_count{node_prefix="<universe_name>", export_type="tserver_export", namespace_name!="system_platform"}[36s])) by (table_name)))[30s:])
   , "quantile", "99", "table_name", "(.+)"
)
```

![Custom query](/images/yp/prometheus-query-editor.png)

> 

Reviewer: ```suggestion

For more information, refer to [Querying Prometheus](https://prometheus.io/docs/prometheus/latest/querying/basics/) in the Prometheus documentation.

```
